<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="bac_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Pesticide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_bttn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 0, 0);
font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Upload Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Tomato Disease Detection and Management System</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_bttn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>560</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 0, 0);
font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Disease Overview</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(118, 118, 118, 153), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Diagnosis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="diagnose_bttn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>190</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 0, 0);
font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Diagnose</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="preventation_txtbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(118, 118, 118, 153), stop:1 rgba(255, 255, 255, 255));

</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="chemicalprev_txtbox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(118, 118, 118, 153), stop:1 rgba(255, 255, 255, 255));
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="diagnosis_txtbox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(118, 118, 118, 153), stop:1 rgba(255, 255, 255, 255));


</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
